CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
 
 ### Features:
-- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview).
-- Added a new variable `QueuedOutgoingRPCWaitTime`. Outgoing RPCs will now be dropped if: more than `QueuedOutgoingRPCWaitTime` time has passed; the worker is never expected to become authoritative in zoning/offloading scenario; the Actor is being destroyed.
-- In local deployments of the Example Project you can now launch Simulated Players in one click. Running `LaunchSimPlayerClient.bat` will launch a single Simulated Player client. Running `Launch10SimPlayerClients.bat` will launch 10.
-- Added an AuthorityIntent component to be used in the future for UnrealGDK code to control loadbalancing.
-- Added support for the UE4 Network Profile to measure relative size of RPC and Actor replication data.
-- Added a VirtualWorkerTranslation component to be used in future UnrealGDK loadbalancing.
-- Added partial framework for use in future UnrealGDK controlled loadbalancing.
-- Add SpatialToggleMetricsDisplay console command.  bEnableMetricsDisplay must be enabled in order for the display to be available.  You must then must call SpatialToggleMetricsDisplay on each client that wants to view the metrics display.
-- Enabled compression in modular-udp networking stack
-- Switched off default rpc-packing. This can still be re-enabled in SpatialGDKSettings.ini
+- We now support Unreal Engine `4.24.3`. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview).
+- We've added a new variable: `QueuedOutgoingRPCWaitTime`. Outgoing RPCs are now dropped in the following three scenarios: more than `QueuedOutgoingRPCWaitTime` time has passed, the worker instance is never expected to become authoritative in zoning/offloading scenario, or the Actor is being destroyed.
+- In local deployments of the Example Project, you can now launch simulated players in one click. Running `LaunchSimPlayerClient.bat` launches a single simulated player client. Running `Launch10SimPlayerClients.bat` launches ten simulated player clients.
+- We've added support for the UE4 Network Profiler to measure relative size of RPC and Actor replication data.
+- We've added a `SpatialToggleMetricsDisplay` console command. You must enable `bEnableMetricsDisplay` in order for the display to be available. You must then must call `SpatialToggleMetricsDisplay` on each client that wants to view the metrics display.
+- We've enabled compression in the Modular UDP networking stack.
+- We've switched off default rpc-packing. You can re-enable this in `SpatialGDKSettings.ini`.
 - Starting a local deployment now checks if the required runtime port is blocked and allows the user to kill it
```

Comment:
What does this mean? _**What**_ checks if the port is blocked, and what is the user allowed to kill? (The process using the port (if there is one))?